### PR TITLE
Search restaurant use case is ready to merge

### PR DESCRIPTION
**Pull request by lisazhong13**
…ce since we need to discuss whether we want that feature or not, also changed the Restaurant entity to make it easier to get the size of reviews for a restaurant.

### CODE REVIEW

File: src/use_case/sortandfilter/SortAndFilterRestaurantInteractor.java
Comment (HarryC493) [inline src/use_case/sortandfilter/SortAndFilterRestaurantInteractor.java:1]: Missing imports from api.

File: src/use_case/sortandfilter/SortAndFilterResaturantInputData.java
Comment (HarryC493) [inline src/use_case/sortandfilter/SortAndFilterResaturantInputData.java:1]: Missing imports from api.

**Review by HarryC493 — CHANGES_REQUESTED** [review]
I have pulled this branch to my local computer and I have noticed that the Interactor and Input Data are missing some necessary imports from the api. For example, SearchSortingMethods and SearchPriceLevel from the Interactor. Please modify these so that the code can compile and run.

**Comment by lisazhong13** [comment]
> I have pulled this branch to my local computer and I have noticed that the Interactor and Input Data are missing some necessary imports from the api. For example, SearchSortingMethods and SearchPriceLevel from the Interactor. Please modify these so that the code can compile and run.

I have imported all the needing modules and they should be fine now

**Pull request merged**